design/fips-compliance.md

```diff
@@ -0,0 +1,199 @@
+# FIPS compliance for conjur-authn-k8s-client
+
+## Background
+
+The National Institute of Standards and Technology (NIST) issued the [FIPS
+ 140-2](https://csrc.nist.gov/publications/detail/fips/140/2/final) 
+Publication Series to coordinate the requirements and standards for cryptography 
+modules that include both hardware and software components.
+
+By making the conjur-authn-k8s-client FIPS compliant, our costumer will gain a 
+cryptographic standard to protect their unclassified, but sensitive data.
```

Comment:
Though you will get into it in the doc, I would maybe add a sentence explaining that how the UX might change

Reply:
the UX won't change at all actually

Reply:
important to note (: